shared/more/style-sheet.render.native.js

```diff
@@ -151,7 +151,27 @@ const Dropdowns = () => (
 // <DropdownDemo/>
 
 const Icons = () => (
-  <Text type='Header'>TODO</Text>
+  <Box style={{...globalStyles.flexBoxColumn}}>
+    {[
+      'computer-big',
+      'computer-bw-m',
+      'fa-custom-copy',
+      'fa-custom-eye',
+      'fa-custom-key',
+      'logo-128',
+      'fa-custom-icon-proof-broken',
+      'fa-custom-icon-proof-good-followed',
+      'fa-custom-icon-proof-good-new',
+      'fa-close',
+      'fa-mobile'
+    ].map(i => [
```

Comment:
:sheep: :sheep: can we type this array to be an array of IconTypes? In case we change one of these in the future flow will tell use to fix this array